driver/SDL/sdl_common.c

```diff
@@ -186,17 +184,21 @@ void mousewheel_handler(SDL_Event * event)
  */
 void keyboard_handler(SDL_Event * event)
 {
+    bool key_state = LV_INDEV_STATE_PRESSED;
     /* We only care about SDL_KEYDOWN and SDL_TEXTINPUT events */
     switch(event->type) {
         case SDL_KEYDOWN:                       /*Button press*/
+        case SDL_KEYUP:                       /*Button release*/
         {
             const uint32_t ctrl_key = keycode_to_ctrl_key(event->key.keysym.sym);
-            if (ctrl_key == '\0')
-                return;
-            const size_t len = strlen(buf);
-            if (len < KEYBOARD_BUFFER_SIZE - 1) {
-                buf[len] = ctrl_key;
-                buf[len + 1] = '\0';
+            
+            if(event->type == SDL_KEYUP)
```

Comment:
This "if" statement can probably be replaced by a simple assignment on the fallthrough `case SDL_KEYDOWN`.

Reply:
I am preassigning the keystone now with a ternary operator.